package.json

```diff
@@ -13,10 +13,10 @@
     "watch": "watch -p 'static/sass/**/*.scss' -c 'yarn run build'"
   },
   "dependencies": {
-    "autoprefixer": "9.4.8",
+    "@canonical/global-nav": "2.0.3",
```

Comment:
Huh? Where did this @canonical come from? Was the package renamed?

Reply:
@nottrobin Yes - it's a package which will add a nav bar filled with Canonical specific links to your page so we moved it under the [@canoncial namepsace](https://www.npmjs.com/package/@canonical/global-nav) last week and deprecated [global-nav](https://www.npmjs.com/package/global-nav)